File: CSX/Functions and Execution Context/invokingFunctions.js
let calls = "";

function jerry(str) {
str = str + "Jerry";
console.log(str);
return kramer(str);
}

function george(str) {
str = str + "George";
console.log(str);
return elaine(str);
}

function elaine(str) {
str = str + "Elaine";
console.log(str);
return str;
}

function kramer(str) {
str = str + "Kramer";
console.log(str);
return george(str);
}

console.log(calls);
calls = jerry(calls);
console.log(calls);

File: CSX - Python/Functions and Execution Context/invoking_functions.py
calls = ''

def jerry(str):
    str += 'Jerry'
    return kramer(str)

def george(str):
    str += 'George'
    return elaine(str)

def elaine(str):
    str += 'Elaine'
    return str

def kramer(str):
    str += 'Kramer'
    return george(str)

calls = jerry(calls)
print(calls)
